### DISCUSSION TITLE
Associated Location-List always starts at the beginning

### DISCUSSION THREAD

**Discussion by rsaur** [sulu/sulu-workshop#34] [Q&A]
Hi,

first of all, thank you all very much for this awesome software  ... I love to work with symfony for years and now I try to get into sulu and start to realize that it is also great software. For the beginning I worked through the workshop and have a Questeion now to lesson 11:  "Add a association between events and locations":
I imported a list of about 100 locations , selected one location for my event which is - lets say - on page 5 of my paginated locations list. I am saving the event, the location gets saved, all is fine. But when I edit the same event again and want to choose a different location, the location list opens at paginated page number 1. Is there a way to open the paginated location list on page 5 where my selected location is?

![screen](https://user-images.githubusercontent.com/61241754/118142924-1ed75300-b40b-11eb-9dfe-f959d35930b6.png)

thanks very much - Rainer

**Comment by niklasnatter** [comment]
Hey, its nice to hear that you like Sulu - thanks for the kind words!
Unfortunately, I dont think it is possible to open the overlay at the page of the selected item at the moment. 

I am afraid that it would be hard to implement such a feature in a performant way. Sulu will load the items of the requested page only to improve performance and decrease the load on the server. As we dont know what page contains the selected item beforehand, we would need to successively load all pages until we find the selected item.

**Reply by rsaur** [reply]
ok, i think, i can live with that ... thanks for your quick reply
